uint8 TAKE_OFF=0
uint8 LAND=1
uint8 HOVER=2
uint8 START_DETECTION=3
uint8 STOP_DETECTION=4
uint8 TRACK_UB=5
uint8 TRACK_FACE=6
uint8 FLY_CIRCLE=7
uint8 RECOGNIZE=8

uint8 CHANGE_CONTROL=9
bool controlX
bool controlYaw
bool controlAltitude

uint8 command


